models.rdf

```diff
@@ -66,6 +66,14 @@
       <rdfs:isDefinedBy rdf:resource="http://pcdm.org/models#"/>
     </rdfs:Class>
 
+    <rdfs:Class rdf:about="http://pcdm.org/models#FileSet">
```

Comment:
I think the key metadata is how the FileSet differs from other FileSets attached to the same Object — this could be creation date, or some (TBD) vocabulary to express which one is preferred or textual/visual/etc. format or something else.  I think I would categorize that as technical metadata, since the purpose is to decide which FileSet to use in a given situation.